src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEnumerations.cs

```diff
@@ -1307,6 +1307,22 @@ public enum BluetoothGattRequestType
         Write = 1,
     }
 
+    /// <summary>
+    /// Enumeration for the GATT service changed type.
```

Comment:
changed -> change, so we have "change type" meaning "type of change", now it can be interpreted as "the type of changed service"

Reply:
type of change is correct! Thanks